Islandora-CLAW/islandora
# Move ldp_contains to rdf_source bundle

Resolves https://github.com/Islandora-CLAW/CLAW/issues/397

How to test:

1. Uninstall Islandora
1. Apply PR
1. Install Islandora

You should now have a ldp_contains field when creating a Rdf Source Fedora Resource, but not when creating a Non Rdf Source Fedora Resource.
